airflow/contrib/operators/sqoop_operator.py

```diff
@@ -126,10 +146,10 @@ def __init__(self,
         self.relaxed_isolation = relaxed_isolation
         self.hcatalog_database = hcatalog_database
         self.hcatalog_table = hcatalog_table
-        # No mutable types in the default parameters
```

Comment:
don't need this in the Operator. Code duplicated in Hook, so it'll take care of the properties field.

Reply:
👍